app/javascript/js/controllers/item_select_all_controller.js

```diff
@@ -63,4 +61,9 @@ export default class extends Controller {
       this.selectAllOverlayTarget.classList.add('hidden')
     }
   }
+
+  // True if there are more pages available and if query encryption run successfully
+  selectAllEnable() {
```

Comment:
I think this should be `selectAllEnabled` (state) not `selectAllEnable` (action).
```suggestion
  selectAllEnabled() {
```